pkg/features/versioned_kube_features.go

```diff
@@ -403,7 +403,7 @@ var defaultVersionedKubernetesFeatureGates = map[featuregate.Feature]featuregate
 
 	ImageMaximumGCAge: {
 		{Version: version.MustParse("1.29"), Default: false, PreRelease: featuregate.Alpha},
-		{Version: version.MustParse("1.30"), Default: true, PreRelease: featuregate.Beta},
```

Comment:
ImageMaximumGCAge was promoted to beta in 1.30, please see https://github.com/kubernetes/kubernetes/pull/123424